src/containers/Tabs/Account.tsx

```diff
@@ -98,7 +101,10 @@ const AccountTabContainer: React.FC<AccountTabContainerProps> = ({ isOpen }) =>
             });
     };
 
-    return (
+    return isOpen ? 
```

Comment:
even though I was on zoom with you when we wrote this, need to refactor it to make it readable.
Maybe ```
const allDataReady = dataReady && adminDataReady
if (isOpen && !allDataReady) {
  return <LoadingPage />
} else if (isOpen && allDataReady) {
  return ...
} else if (!isOpen) {
  return ...
}```